client/integrations/integrations.go

```diff
@@ -804,6 +810,120 @@ func getVersionId(name string, filter string) (version string, err error) {
 	}
 }
 
+// ExportWithThreads exports all Integration Flows in the specified folder using a configurable number of threads
```

Comment:
I don't think "threads" is the right word. To my understanding goroutines != thread. How about `ExportConcurrent`? And `numConnections`?

Reply:
make sense..